fe/src/main/java/org/apache/doris/analysis/ShowProcStmt.java

```diff
@@ -82,10 +82,6 @@ public ShowResultSetMetaData getMetaData() {
 
     @Override
     public RedirectStatus getRedirectStatus() {
```

Comment:
I think rename the `forward_to_master` to `show_stmt_forward_to_master`, and change the default value to true is a better way.

**All show stmt should have the consistent behavior.**